src/libraries/Microsoft.PowerFx.Core/Parser/ParseFormulasResult.cs

```diff
@@ -94,12 +94,14 @@ public UDF(IdentToken ident, IdentToken returnType, HashSet<UDFArg> args, TexlNo
     internal class UDFArg
     {
         internal IdentToken VarIdent;
-        internal IdentToken VarType;
+        internal IdentToken VarType;

```

Comment:
naming nit: why start with Var?

Also, both are Idents. Maybe instead call these NameIdent and TypeIdent?